src/components/MarketWidget/MarketsWidgetHeader.tsx

```
@@ -84,6 +102,242 @@ const setLocalStorageItem = <T,>(key: string, value: T): void => {
+      });
+    }
+  }
+  
+  // Update visibility and store it
+  const toggleVisibility = (columnId: string) => {
+    const newVisibility = { ...columnVisibility };
+    newVisibility[columnId] = !newVisibility[columnId];
+    
+    // Don't allow hiding the pair column
+    if (columnId === 'pair') {
+      delete newVisibility['pair'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)